app/lib/katello/http_resource.rb

```diff
@@ -137,13 +137,8 @@ def delete(a_path = nil, headers = {})
 
       # re-raise the same exception with nicer error message
       def raise_rest_client_exception(e, a_path, http_method)
-        # message method in rest-client is hardcoded - we need to override it
         msg = "#{name}: #{e.message} #{e.http_body} (#{http_method} #{a_path})"
-        (class << e; self; end).instance_eval do
```

Comment:
I guess this was written in an age when the RestClientException didn't have the accessor for message...